.travis.yml

```diff
@@ -50,7 +50,12 @@ addons:
     packages: ninja-build libstdc++-5-dev
 
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install ninja; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+      wget -q https://github.com/ninja-build/ninja/releases/download/v1.7.2/ninja-mac.zip;
```

Comment:
What's the working directory here? No need to generate and use a temp path?

Reply:
I *think* the working directory is the directory where the repo is downloaded in the Travis VM. I don't think it's necessary to make a temp directory. I just want to put `ninja` on the PATH. It'll all be deleted once the bot is done building.